hilti/include/ast/scope-lookup.h

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2020 by the Zeek Project. See LICENSE for details.
+//
+// Functionality factored out from scope.h to avoid header loops.
+
+#pragma once
+
+#include <optional>
+#include <string>
```

Comment:
Let's move the include of `<utility>` for `pair` from the `.cc` file here.

```suggestion
#include <utility>
```